Dynavity/Dynavity/persistence/data-mapping/CanvasDTO.swift

```diff
@@ -0,0 +1,23 @@
+struct CanvasDTO: Mappable {
+    typealias Model = Canvas
```

Comment:
What do these aliases do? They don't seem to be used in the rest of the file, and the project seems to build properly even when commented out.